src/middleware.ts

```diff
@@ -15,18 +16,16 @@ export const config = {
   ],
 }
 
-const TENANTS: { slug: string; domain: string; id: string }[] = [
-  { slug: 'nwac', domain: 'nwac.us', id: '1' },
-  { slug: 'sac', domain: 'sierraavalanchecenter.org', id: '2' },
-  { slug: 'snfac', domain: 'sawtoothavalanche.com', id: '3' },
-]
+const PRODUCTION_TENANTS = getProductionTenantSlugs()
 
-const REDIRECT_TO_CUSTOM_DOMAIN =
-  process.env.REDIRECT_TO_CUSTOM_DOMAIN?.toLowerCase() === 'true' ||
-  process.env.NODE_ENV === 'production'
+const TENANTS: { slug: string; customDomain: string; id: string }[] = [
+  { slug: 'nwac', customDomain: 'nwac.us', id: '1' },
```

Comment:
We need the ID because of how Payload sets the cookie, right? If we end up doing the "multi-tenant admins on a tenant-scoped site only see content for one tenant" do we need to do the cookie handling ourselves again? Only reason I ask is - we may need to just pre-populate tenants in the DB and make this list have them all, otherwise registering a new tenant (and generating its ID) will require a redeploy as we change this as well.